### DISCUSSION TITLE
TypeError: 'numpy._DTypeMeta' object is not subscriptable

### DISCUSSION THREAD

**Discussion by torikxx** [OpenTalker/SadTalker#714] [Q&A]
Dear GitHub community,

I am reaching out to you seeking assistance with an issue I have encountered while trying to launch version 0.0.2 rc of the program. Upon execution, I encounter an error that leads to an unsuccessful program run:

NumPyArrayGeneric = numpy.ndarray[typing.Any, numpy.dtype[numpy.generic]]
TypeError: 'numpy._DTypeMeta' object is not subscriptable

I have attempted to reinstall and update the NumPy library, change versions, reinstall Python, but none of these measures have been successful. Given your knowledge or experience with this issue, I would greatly appreciate any help or advice you can provide.

**Comment by DennisPeeters** [comment]
same issue
